lifx_control_panel/frames.py

```diff
@@ -584,7 +599,7 @@ def get_color_from_palette(self):
         if color:
             # RGBtoHBSK sometimes returns >65535, so we have to truncate
             hsbk = [min(c, 65535) for c in lifxlan.RGBtoHSBK(color, self.hsbk[3].get())]
-            self.set_color(hsbk)
+            self.set_color(hsbk);
```

Comment:
Was there a reason for a semicolon here?

Reply:
only my habits as a web developer 👀